soc/arm/nuvoton_npcx/npcx7/linker.ld

```diff
@@ -0,0 +1,35 @@
+/* linker.ld - Linker command/script file */
+
+/*
+ * Copyright (c) 2020 Nuvoton Technology Corporation.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/* Temporary solution to put npcx header in front of ec image. */
+#define CONFIG_RO_HDR_SIZE	0x40
+
+MEMORY
+{
+    /*
+     * Header structure used by npcx booter (ROM code).
+     * Please notice the location of header must be in front of FW
+     * which needs copy. But header itself won't be copied to code ram
```

Comment:
what does 'needs copy' mean? This is a bit opaque to me

Reply:
The firmware of npcx ec isn't executed on embedded spi flash directly. The rom code copies it from flash to code ram by the header. Then set pc to the reset vector address which also indicated by header.